jeto/controllers.py

```diff
@@ -401,6 +402,9 @@ def get_git_references(projectId):
 api.add_resource(UserApi, '/api/users', endpoint='users')
 api.add_resource(UserApi, '/api/users/<id>')
 
+api.add_resource(SSLApi, '/api/SSLKeys', endpoint='SSLKey')
```

Comment:
Conversation on irc : 
We should see sslkey as "ssl keystore", not as "ssl .key file".